src/schemas/MessageSchema.js

```diff
@@ -72,30 +79,16 @@ module.exports = graphql.buildSchema(`
   type Message {
     edges: [String],
     messageid: ID,
-    createdtime: String,
+    eventtime: String,
+    sourceeventid: String,
     sentiment: Float,
+    entities: [String],
     title: String,
-    originalSources: [String],
-    sentence: String,
+    externalsourceid: String,
```

Comment:
This will also require some changes in project-fortis-interfaces. Would be ideal to have them before merging this PR so that we can match up things and review related changes at the same time.